tests/contact-callbacks.js

```diff
@@ -0,0 +1,80 @@
+const test = require('ava')
+const createUnitBox = require('./helpers/create-unit-box.js')
+const loadAmmo = require('./helpers/load-ammo.js')
+
+const CF_CUSTOM_MATERIAL_CALLBACK = 8
+const DISABLE_DEACTIVATION = 4
+
+
+function shouldTest(t, Ammo) {
+  if (Ammo.addFunction) return true
+  t.log(`Passed '${t.title}' without testing (requires ammo with addFunction support)`)
+  t.pass()
```

Comment:
@kripken This `pass` pretty ugly, I am going to see if we can do some dynamic skipping in upstream ava.

Reply:
Sounds good.